broker/lib/fabrik/DirectorInstance.js

```diff
@@ -50,9 +50,15 @@ class DirectorInstance extends BaseInstance {
   }
 
   get async() {
```

Comment:
i think this file might need major refactoring once rate limiting PR goes in.

Reply:
The changes were merged and verified by @abh1kg